src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -331,6 +331,15 @@ public virtual StringDataFrameColumn Info()
         /// </summary>
         public virtual DataFrameColumn Description() => throw new NotImplementedException();
 
+        /// <summary>
+        /// Returns the indices of non-null values that, when applied, result in this column being sorted in ascending order. Also returns the indices of null values in <paramref name="nullIndices"/>.
+        /// </summary>
+        /// <param name="nullIndices">Indices of values that are <see langword="null"/>.</param>
+        internal virtual PrimitiveDataFrameColumn<long> GetAscendingSortIndices(out Int64DataFrameColumn nullIndices) => throw new NotImplementedException();
+
+        /// <summary>
+        /// Returns the indices of non-null values that, when applied, result in this column being sorted in ascending order
+        /// </summary>
         internal virtual PrimitiveDataFrameColumn<long> GetAscendingSortIndices() => throw new NotImplementedException();
```

Comment:
What do we do with the `null` values in this API? Are they just dropped? When is that valuable?

Reply:
Yup, they are just dropped. At the moment it's used for `Median`. We get the ascending sort indices and use that to calculate the median. I don't have strong opinions about keeping/removing this API. My reasoning to keep it was that I didn't want to break anyone who's overriding this API, so I made a new one.

Reply:
> My reasoning to keep it was that I didn't want to break anyone who's overriding this API,

It's `internal`.

Reply:
Huh, my bad. For some reason I thought it was `protected`. Fixed now. I added the `nullIndices` as an out parameter to the existing method.